app/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    "arrowParens": "always",
```

Comment:
options here is subject for discussion. See https://prettier.io/docs/en/options.html for details

Reply:
updated according to discussion